support-frontend/webpack.common.js

```diff
@@ -83,6 +84,10 @@ module.exports = (cssFilename, jsFilename, minimizeCss) => ({
 		}),
 		...(minimizeCss ? [new CssMinimizerPlugin()] : []),
 		new CleanUpStatsPlugin(),
+		new CircularDependencyPlugin({
+			// exclude detection of files based on a RegExp
+			exclude: /a\.js|node_modules/,
```

Comment:
What's `a.js`?

Reply:
No idea, that came from the example in the docs so I didn't look too closely at it. I assumed until you pointed it out that it was filtering all .js files but now I see it isn't and I'm just as confused. I'll delete it, see what happens